src/Sql/dbo/Stored Procedures/Collection_ReadWithGroupsAndUsersByUserId.sql

```diff
@@ -0,0 +1,30 @@
+CREATE PROCEDURE [dbo].[Collection_ReadWithGroupsAndUsersByUserId]
+	@UserId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+	
+    DECLARE @TempUserCollections TABLE(Id UNIQUEIDENTIFIER, OrganizationId UNIQUEIDENTIFIER, Name VARCHAR(MAX), CreationDate DATETIME2(7), RevisionDate DATETIME2(7), ExternalId NVARCHAR(300), ReadOnly BIT, HidePasswords BIT)
```

Comment:
small nit: Could we possibly put the columns on new line breaks, to make this a bit easier to read?